cmd/mindthegap/push/bundle/bundle.go

```diff
@@ -361,7 +405,7 @@ func pushImages(
 	regNames := cfg.SortedRegistryNames()
 
 	eg, egCtx := errgroup.WithContext(context.Background())
-	eg.SetLimit(imagePushConcurrency)
+	eg.SetLimit(max(imagePushConcurrency, 1))
```

Comment:
Why is this needed?

Reply:
https://pkg.go.dev/golang.org/x/sync/errgroup#Group.SetLimit
>A limit of zero will prevent any new goroutines from being added.

Actually I misunderstood what this is for, let me make that clearer in code.